wikimedia/wikipedia-ios
# Widget 0xdead10cc fixes part III: it was the old widgets all along

**Fixes Phabricator ticket:** 
https://phabricator.wikimedia.org/T263247

### Notes
* Updates old widgets to use new widget fetching method
* Fixes retain cycles in image fetching exposed by image fetching via the `UIImageView` extension in the old widgets

### Test Steps
1. Simulator > Erase all content and settings
2. Restart Xcode
3. Set a breakpoint on [MWKDataStore shared]
4. Run the old widget targets
5. Ensure the breakpoint isn't hit
6. Inspect the memory graph
7. Ensure there are no data stores or `NSManagedObjectContext`s remaining
